app/helpers/application_helper/toolbar/vm_clouds_center.rb

```diff
@@ -25,7 +25,8 @@ class ApplicationHelper::Toolbar::VmCloudsCenter < ApplicationHelper::Toolbar::B
           :url_parms => "main_div",
           :confirm   => N_("Perform SmartState Analysis on the selected items?"),
           :enabled   => false,
-          :onwhen    => "1+"),
+          :onwhen    => "1+",
+          :klass     => ApplicationHelper::Button::VmInstanceScan),
```

Comment:
[wip] last spec is failing because of this added class, still can't understand why though...